### DISCUSSION TITLE
Support for more than 1 custom domain?

### DISCUSSION THREAD

**Discussion by schmidna** [microsoft/AzUrlShortener#315] [Q&A]
With a quick browse I'm unsure if one could utilize more than one custom domain with this solution in it's current form - can you? 

The use case I have in mind is for an organization where shortening Urls for external users would be a huge help but equally so for internal and I'd like to use two separate domains to achieve it. In theory I could maybe use sub-domains but that's not ideal.

Awesome project!!!

**Comment by fboucher** [comment]
hmmm.
Maybe using cloudflare it could be possible.
Or you could have 2 Azure Function pointing to the same Table storage. I do this often to deploy and test beta version (without having to populate ton of data)
